scenes/src/app/scenes/radio/radio-scene.scss

```diff
@@ -0,0 +1,18 @@
+.scene-radio-group {
+  display: flex;
+  flex-direction: column;
+  margin: 10px 0;
+}
+
+.scene-radio-button {
+  margin: 5px;
+}
+
+.scene-radio-title {
+  text-align: center;
+}
+
+.scene-radio-icon {
+  font-size: 45px;
+  color: rgba(red, 0.6);
```

Comment:
Is this hardcoded color going to be an issue if the hue changes?

Reply:
All of the examples are actually red, but then we use a CSS filter to shift the hue, so I think this should be ok